tests/Shiny.Prism.Tests/Tests/PrismStartupTests.cs

```diff
@@ -53,6 +58,27 @@ public void ExpectedTypesAreRegisteredAndResolve(Type serviceType, Type implemen
             Assert.Same(fromShiny, PrismContainerExtension.Current.Resolve(serviceType));
         }
 
+        [Fact]
+        public void CanResolveAll()
+        {
+            ShinyPrismTestHost.Init(_testOutputHelper, services =>
+            {
+                services.AddSingleton<IPushDelegate, MockPushDelegate>();
+            });
+
+            Assert.True(PrismContainerExtension.Current.IsRegistered(typeof(IPushDelegate)));
+
+            var fromShiny = ShinyHost.Resolve<IPushDelegate>();
+            Assert.IsType<MockPushDelegate>(fromShiny);
+            Assert.Same(fromShiny, PrismContainerExtension.Current.Resolve(typeof(IPushDelegate)));
+            Assert.Same(fromShiny, ShinyHost.Container.Resolve<IPushDelegate>());
+
+            Assert.Equal(new[] { fromShiny }, PrismContainerExtension.Current.Resolve<IEnumerable<IPushDelegate>>());
```

Comment:
Note the test is failing on Line 77. This `Resolve` works as expected, so the issue has to be somewhere in the Shiny integration.